src/main/include/controllers/ValorController.h

```diff
@@ -272,5 +276,4 @@ class ValorController
 

```

Comment:
Can we add the function definition for initSendable and make it pure virtual? That way every ValorController implementation has to implement initSendable, and makes sure every programmer is being a good developer by adding logging.